decksite/scrapers/decklist_test.py

```diff
@@ -165,3 +166,97 @@ def test_parse5():
     print(d)
     assert len(d['maindeck']) == 13
     assert len(d['sideboard']) == 1
+
+# Test that a 71 card deck includes the last 15 as sideboard
+def test_parse6():
+    s = """
+        4 Animist's Awakening
+        4 Copperhorn Scout
+        15 Forest
+        4 Harvest Season
+        4 Jaddi Offshoot
+        4 Krosan Wayfarer
+        4 Loam Dryad
+        4 Nantuko Monastery
+        4 Nest Invader
+        4 Quest for Renewal
+        4 Rofellos, Llanowar Emissary
+        2 Sky Skiff
+        4 Throne of the God-Pharaoh
+        1 Distress
+        2 Dystopia
+        1 Infest
+        4 Memoricide
+        2 Nature's Ruin
+        1 Pharika's Cure
+        4 Scrabbling Claws
+    """
+    s = textwrap.dedent(s)
+    d = decklist.parse(s)
+    print(d)
+    assert sum(d['maindeck'].values()) == 61
+    assert len(d['maindeck']) == 13
+    assert len(d['sideboard']) == 7
+
+# Test a 63 card deck + 12 sideboard
```

Comment:
I want to smack whoever does this :)

Reply:
Haha, well I wanted to test that when adding to the sideboard bottom up, it actually stopped when adding the N cards from a row would exceed 15